src/doc/cli_manual/contributing.rst

```diff
@@ -1,37 +1,35 @@
 Contributing To VisIt CLI Documentation
 =======================================
 
-**Note: This procedure is planned for change in the future.**
+.. note::
+   We are still refining this procedure!
 
-At present, all VisIt Python CLI documentation is actually composed directly
-as Python strings in the source C++ file in ``../../visitpy/common/MethodDoc.C``.
+The Python doc strings for most functions in VisIt's cli are generated
+from the examples embedded in the ``cli_manual/functions.rst`` file.
+This allows us to have a single source for both our CLI sphinx docs 
+and the doc strings embedded in VisIt's compiled Python module. 
+The ``functions_to_method_doc.py``
```

Comment:
This paragraph ends a little awkardly...

Reply:
good catch, I added a bit more text.